docs/changelog.mdx

```diff
@@ -7,6 +7,16 @@ This is the changelog for the Testkube Control Plane, latest releases are on top
 - **Patch releases** focus on critical bugs and security issues.
 :::
 
+
+## Patch Release v1.15.3 (2025-06-04)
+
+This release is available in Helm Chart v2.321.3
+
+### Improvements
+
+- Fixes issues with artifact previews. Sometimes they were not rendering artifacts correctly.
+- Fix newCredential is unknown function when creating a new Test Workflow template.
```

Comment:
```suggestion
- Fixed an issue where Workflow Templates could not be created due to unknown function `new credentials`
```

Another recommendation is to start with the important part "workflows could not be created" and the details afterwards.